docs/dictionary-and-concepts/dictionary-charges.md

```diff
@@ -10,32 +10,42 @@ The terms listed here are all terms relating to the topics of defining charges.
 |----------|-------------|
 | **A** |   |
 | **B** |   |
+| Business Reason Code | Indicates the business context, e.g. D10 = 'Request update charge information', which covers all three charge operation types; 'Create', 'Update' and 'Stop' |
 | **C** |   |
-| Charge Name  | The identifyer of a specific charge  |
-| Charge Occurrence | An indication on how often the charge is being billed  |
-| Charge Owner | An identifier for the charge owner  |
-| Charge Type | This could be either a Tariff, Fee or Subscription  |
-| Currency Unit | Defining which currency is used  |
+| Charge Description |  |
+| Charge Id | A charge identifier provided by the Market Participant. Combined with Charge Owner it becomes unique |
+| Charge Name |  |
+| Charge Owner | A charge owner identification, e.g. the Market Participant's GLN or EIC number |
+| Charge Type | The type of charge; tariff, fee or subscription  |
+| Currency | The charge price currency |
 | **D** |   |
 | **E** |   |
 | **F** |   |
 | **G** |   |
 | **H** |   |
 | **I** |   |
+| Industry Classification | Indicates the industry context, e.g. E23 = Electricity |
```

Comment:
Is E23 an eBIX name? Do we use that?

Reply:
E23 is an ebIX value. And yes we will still be using those :)

Reply:
Ok. Thx.